src/natcap/invest/urban_nature_access.py

```diff
@@ -8,11 +8,11 @@
 import numpy.testing
 import pygeoprocessing
 import taskgraph
+from natcap.invest import MODEL_METADATA
```

Comment:
Hey @phargogh , this looks fine to me. This PR is just to make the other review cleaner to review, but I wanted to note there are some UNA chances here, but they are simple and straight forward, so it looks good me to me!

Reply:
Yep, I had to touch those because of a couple of merge conflicts that were causing other tests to fail.  Sorry for not mentioning it in the PR description!